StackOverflow/Model/DataFetchDelegate.h

```diff
@@ -0,0 +1,16 @@
+//
+//  DataFetchDelegate.h
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol DataFetchDelegate <NSObject>
```

Comment:
I'm not sure why we actually need two protocols, surely one would have been sufficient ?